src/main/java/com/powsybl/openloadflow/sa/AcSecurityAnalysis.java

```diff
@@ -230,4 +312,96 @@ private PostContingencyResult runPostContingencySimulation(LfNetwork network, Ac
     }
+
+    private Optional<OperatorStrategyResult> runActionSimulation(LfNetwork network, AcLoadFlowContext context, Contingency contingency, OperatorStrategy operatorStrategy,
+                                                                 LimitViolationManager preContingencyLimitViolationManager,
+                                                                 SecurityAnalysisParameters.IncreasedViolationsParameters violationsParameters,
+                                                                 LimitViolationsResult postContingencyLimitViolations, Map<String, LfAction> lfActionById,
+                                                                 PreContingencyNetworkResult preContingencyNetworkResult, boolean createResultExtension,
+                                                                 List<String> allSwitchesToCloseIds) {
+        OperatorStrategyResult operatorStrategyResult = null;
+
+        if (checkCondition(operatorStrategy, postContingencyLimitViolations)) {
+            LOGGER.info("Start operator strategy {} after contingency '{}' simulation on network {}", operatorStrategy.getId(),
+                    operatorStrategy.getContingencyId(), network);
+
+            List<LfAction> operatorStrategyLfActions = operatorStrategy.getActionIds().stream()
+                    .map(id -> {
+                        LfAction lfAction = lfActionById.get(id);
+                        if (lfAction == null) {
+                            throw new PowsyblException("Action '" + id + "' of operator strategy '" + operatorStrategy.getId() + "' not found");
+                        }
+                        return lfAction;
+                    })
+                    .collect(Collectors.toList());
+            // FIXME to remove when connectivity will be fixed
+            operatorStrategyLfActions.stream().limit((long) operatorStrategyLfActions.size() - 1).forEach(LfAction::apply);
+            operatorStrategyLfActions.get(operatorStrategyLfActions.size() - 1).apply(true, allSwitchesToCloseIds); // the last apply compute the final connectivity.
+
+            Stopwatch stopwatch = Stopwatch.createStarted();
+
+            // restart LF on post contingency and post actions equation system
+            context.getParameters().setVoltageInitializer(new PreviousValueVoltageInitializer());
+            AcLoadFlowResult postActionsLoadFlowResult = new AcloadFlowEngine(context)
+                    .run();
+
+            boolean postActionsComputationOk = postActionsLoadFlowResult.getNewtonRaphsonStatus() == NewtonRaphsonStatus.CONVERGED;
+            // FIXME: to be checked.
+            var postActionsViolationManager = new LimitViolationManager(preContingencyLimitViolationManager, violationsParameters);
+            var postActionsNetworkResult = new PostContingencyNetworkResult(network, monitorIndex, createResultExtension, preContingencyNetworkResult, contingency);
```

Comment:
I don't think this is correct to use PostContingencyNetworkResult, flow transfer will still be calculated for contingency. What do we really want here?

Reply:
@sylvlecl and @EtienneLt : good question !

Reply:
Flow transfer will be at NaN after actions.

Reply:
Ok make sense